### ISSUE TITLE
Unusual interactions with Cogmap 1 HoS locker

### ISSUE THREAD

**Issue by FlameArrow57** [goonstation/goonstation#18663]
### Labels

[BUG]

### Description

Title

### Steps to reproduce

1. Stand at X
![image](https://github.com/goonstation/goonstation/assets/53062374/df2c83ef-9e64-46c5-b1d1-8e57839df82d)
2. Right click locker location
3. Do Pick up right click menu interaction with something near the bottom of the stack (HoS cape or HoS clothing box)
4. The click is instead routed to the door, and does not pick up what you did the right click pick up on

### Expected Behavior

For the item to be picked up, not the door to be hand attacked

**Comment by Sbmhawk** [comment]
I think that's a problem with the pickup verb, I believe it gets the first thing in the list that has the same first word.

If you use it on your organ/limb, you pat yourself instead of picking up it, unless that was fixed

**Comment by Tarmunora** [comment]
Yeah, it's an BYOND-side issue with apostrophes in names fucking with verbs. I'm not sure we could even fix this short of nuking apostrophes
